nilearn/surface/tests/test_surface.py

```diff
@@ -381,7 +382,18 @@ def test_load_uniform_ball_cloud():
             assert_equal(len(w), 0)
     assert_warns(surface.EfficiencyWarning,
                  surface._load_uniform_ball_cloud, n_points=3)
-    for n_points in [3, 10, 20]:
+    for n_points in [3, 7]:
+        computed = surface._uniform_ball_cloud(n_points)
+        loaded = surface._load_uniform_ball_cloud(n_points)
+        assert_array_almost_equal(computed, loaded)
+    if sklearn.__version__ > '0.21':
+        # results of scikit-learn kmeans change after
+        # https://github.com/scikit-learn/scikit-learn/pull/9288
+        # (commit e8f2708e05a9982e86320d994abcbcf02a1536d9)
+        # so computed points will differ from the precomputed ones shipped with
+        # nilearn
+        return
```

Comment:
if I understand correctly, we'll precompute other points when we no longer support sklearn <= 0.21 ?